### ISSUE TITLE
Suggestion: Shipping binaries for python with PyInstaller

### ISSUE THREAD

**Issue by omars44** [flatpak/flatpak-builder-tools#350]
I think it might be a good idea to depend on PyInstaller package to generate binaries that has python and deps already embedded.

Less manifests to worry about, so can contribute to a shorter release cycle, or faster workflow

Any thoughts on that? I'm experimenting with that route, and if there is interest, I can automate the process somehow, or at least write guidelines in the docs

I'm really fond of the flatpak ecosystem, just not an expert on the topic, so any feedback is appreciated.

Cheers!

ref: https://pyinstaller.org/en/stable/index.html

**Comment by TingPing** [comment]
If you would like to create a new tool to add to this repo then it can be merged. I'm not sure its helpful to have open ended suggestions. This repo isn't really structured that way.

**Comment by omars-code** [comment]
@TingPing thanks, will see

**Comment by JakobDev** [comment]
You mean, you like to build a binary with PyInstaller outside the Flatpak and just copy it in the Flatpak?

**Comment by hfiguiere** [comment]
I don't think this is in scope. These are toosl to generate dependency lists from requirements.

**Comment by jamesdbrock** [comment]
Hi @omars44 I think that's a really good idea. Did you make any progress building a Flatpak from a Pyinstaller dist?
